<launch>

  <node pkg="line_detection" name="threshold" type="threshold.py" output="screen">
    <param name="subscriber_image_topic" type="str" value="/camera/image_raw/compressed" />
    <param name="publisher_image_topic" type="str" value="/line_image/compressed" />
    <param name="buffer_size" type="int" value="52428800" />
    <rosparam command="load" file="$(find line_detection)/roi.yaml" />

  </node>

  <node pkg="line_detection" name="pc_publisher" type="pointcloud_publisher.py" output="screen">
    <param name="subscriber_image_topic" value="/threshold/line_image/compressed" />
    <param name="buffer_size" type="int" value="52428800" />
<!--
    <rosparam command="load" file="$(find line_detection)/roi.yaml" />
-->
  </node>
<!--
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />
-->
</launch>
